### PR TITLE
OCPBUGS-14949: Clarify install-config.yaml changes

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OCPBUGS-14949

Link to docs preview: https://62136--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-preparing-to-install-sno.html#install-sno-requirements-for-installing-on-a-single-node_install-sno-preparing

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by gpei** [comment]
@amolnar-rh 
> In all cases, you must change the `platform.none: {}` parameter in the `install-config.yaml` configuration file to specify the platform.
This still sounds a bit confused to me that there's no `platform.none: {}` in the `install-config.yaml` if we directly installing on AWS or vSphere platforms.

so how about dropping this sentence directly?
`In all cases, you must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.`

**Comment by amolnar-rh** [comment]
@gpei Removed it with the following "For example, ..." sentence, too.

File: modules/install-sno-requirements-for-installing-on-a-single-node.adoc [hunk 1]
```diff
@@ -12,9 +12,9 @@ Installing {product-title} on a single node alleviates some of the requirements
 
 * *CPU Architecture:* Installing {product-title} on a single node supports `x86_64` and `arm64` CPU architectures.
 
-* *Supported platforms:* Installing {product-title} on a single node is supported on bare metal, vSphere, AWS, and Red Hat OpenStack platforms. In all cases, you must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
+* *Supported platforms:* Installing {product-title} on a single node is supported on bare metal, vSphere, AWS, Red Hat OpenStack, and Red Hat Virtualization platforms.
```
Comment (jab-rh) [inline modules/install-sno-requirements-for-installing-on-a-single-node.adoc:15]: I think we can use one of

```
:rh-virtualization-first: Red Hat Virtualization (RHV)
:rh-virtualization: RHV
```

here in place of the text.

**Review by jab-rh — COMMENTED** [review]
@amolnar-rh, left one comment, thanks!

**Review by sheriff-rh — COMMENTED** [review]
LGTM, merging.

**Comment by sheriff-rh** [comment]
@amolnar-rh looks like a conflict in 4.13, can you investigate and cherry pick manually?

**Pull request merged**